docker/helk_install.sh

```diff
@@ -21,11 +21,12 @@ INSTALL_MINIMUM_MEMORY=5000
 ## In MBs
 INSTALL_MINIMUM_MEMORY_NOTEBOOK=8000
 ## In GBs
-INSTALL_MINIMUM_DISK=20
+INSTALL_MINIMUM_DISK=25
```

Comment:
need wiki update if this is increased to 25. also, any reason from 20 to 25? I am OK with this but just wanted to know for sure.

Reply:
No there is no particular reason. It is a mistake of mine. I will fix that later tonight.